experimental/src/main/scala/org/testcontainers/testcontainers4s/containers/PostgreSQLContainer.scala

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.testcontainers4s.containers
+
+import org.testcontainers.containers.{PostgreSQLContainer => JavaPostgreSQLContainer}
+
+object PostgreSQLContainer {
+
+  val defaultDockerImageName = s"${JavaPostgreSQLContainer.IMAGE}:${JavaPostgreSQLContainer.DEFAULT_TAG}"
+  val defaultDatabaseName = "test"
+  val defaultUsername = "test"
+  val defaultPassword = "test"
+
+  // TODO: add extraConfiguration with java container?
+  case class Def(
+    dockerImageName: String = defaultDockerImageName,
```

Comment:
there is a huge problem with this style of API: you have to duplicate every property we introduce in Java, and it confuses users when they see something in Java, but not in Scala.

E.g. how do I set the TmpFS option here?
If I understand the implementation correctly, one cannot even access the `JavaPostgreSQLContainer` instance to apply the modifications.

Reply:
The current version of scala API uses the same approach. Yes, sometimes scala version could be a bit outdated. But you always can use `underlyingUnsafeContainer` in the started container and call any method from the original java container.